src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+const Person = ({ person }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+      {(person.isMarried && person.sex === 'm')
+      && <p className="Person__partner">{`${person.partnerName} is my wife`}</p>}
+      {(person.isMarried && person.sex === 'f')
+      && <p className="Person__partner">{`${person.partnerName} is my husband`}</p>}
+      {!person.isMarried
+      && <p className="Person__partner">I am not married</p>}
```

Comment:
Now all these checks are very similar, try to combine them and put them in a separate variable, and then use this variable here